### PR TITLE
update: Close detail ( t ) mtt documentation windows and debug.log doesnt show in cmd

### PR DESCRIPTION
**Pull request by Briancoughlin**
**Select the type of change:**

- [x] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**

Windows Cmd does not display live information in a similar way to Mac terminal
>

**Issue Number:** 

MTTDOC-208. and #163  and #170

### CODE REVIEW

**Review by LukeStampfli — DISMISSED** [review]
Just a small change needed to the path I think. Else it looks fine

File: versioned_docs/version-0.1.0/tutorials/goldenpath.md [hunk 1]
```diff
@@ -256,10 +260,30 @@ For Windows you should do the following:
   <path to project>/Build/GoldenPath.exe -mlapi client
   ```
 
-:::note
-You may get a UAC prompt requesting prermission for the binary to run you should allow it.
+:::important
+On Windows, no standard out stream exists by default, so you will need to view the Debug.log file to see the outputs. You can find the Debug.log  files in:
+C:\Documents and Settings\User_Name\Local Settings\Application Data\Unity\Editor
```
[the remark this answers is not in the document]
Comment (LukeStampfli) [reply versioned_docs/version-0.1.0/tutorials/goldenpath.md:265]: the `CompanyName` should default to `DefaultCompany` for a new project and the `product name` should be equal to the project's name.

**Comment by lkrell** [comment]
Make sure to also update the same file in /docs for Develop version.

File: versioned_docs/version-0.1.0/tutorials/goldenpath.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Briancoughlin) [reply versioned_docs/version-0.1.0/tutorials/goldenpath.md]: now updated

**Pull request merged**